packages/node_modules/@node-red/nodes/core/core/80-function.html

```diff
@@ -27,7 +27,7 @@
             name: {value:""},
             func: {value:"\nreturn msg;"},
             outputs: {value:1},
-            noerr: {value:0,required:true,validate:function(v) { return ((!v) || (v === 0)) ? true : false; }}
+            noerr: {value:0,required:true,validate:function(v) { return ((!v) || (v === 0)); }}
```

Comment:
a simple `return !v` should do it as on v=0 `!v` already validates to true.

Reply:
You're correct. Removed.